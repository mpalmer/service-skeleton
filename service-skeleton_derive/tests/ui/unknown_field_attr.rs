#![allow(unused_imports, unused_crate_dependencies)]
use service_skeleton_derive::ServiceConfig;

#[derive(ServiceConfig)]
struct Config {
	#[config(something)]
	foo: bool,
}

fn main() {}
