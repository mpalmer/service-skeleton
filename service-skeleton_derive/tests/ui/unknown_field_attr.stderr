error: Unknown field: `something`
 --> tests/ui/unknown_field_attr.rs:6:11
  |
6 |     #[config(something)]
  |              ^^^^^^^^^
